packages/xo-server/src/xo-mixins/xen-servers.mjs

```diff
@@ -371,6 +385,30 @@ export default class {
           return xapiObjectToXo(object, dependents)
         }
 
+        const markPool = async () => {
+          const now = Date.now()
+          const { pool } = xapi
+
+          try {
+            await asyncEach(Object.entries(pool.other_config), ([key, value]) => {
+              if (key.startsWith(poolMarkingPrefix)) {
+                const { lastConnected } = JSON.parse(value)
+                if (now - lastConnected > poolMarkingMaxAge) {
+                  return pool.update_other_config(key, null)
+                }
+              }
+            })
+
+            await pool.update_other_config(
+              poolMarkingPrefix + this._applianceUuid,
+              JSON.stringify({ lastConnected: now })
+            )
```

Comment:
Do we also want XOs built from the sources to add the entry to the other config with a `xo:appliance:` key? Since `this._applianceUuid` will be `''`.

Reply:
Ideally, at some point, knowing if it's an official XOA or not will be interesting. But on the short term, all XOs should be listed (XO from sources or not).

Reply:
Hmm, indeed, I should figure what key to use for XO from sources :thinking: 

Any ideas?

Reply:
Same than my previous comment, let's discuss about it for 30min